python/src/FunctionalChaosAlgorithm_doc.i.in

```diff
@@ -280,6 +411,6 @@ residual : float
 Parameters
 ----------
 strategy : :class:`~openturns.ProjectionStrategy`
```

Comment:
```suggestion
projectionStrategy : :class:`~openturns.ProjectionStrategy`
```
For the sake of consistency:
![image](https://github.com/openturns/openturns/assets/31989332/ed205376-6f2b-4679-bee2-86b04275c974)